tests/vm_test.go

```diff
@@ -1188,6 +1188,100 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			Expect(strings.HasPrefix(stdErr, "Error from server (NotFound): virtualmachines.kubevirt.io")).To(BeTrue(), "should fail when deleting non existent VM")
 		})
 
+		Context("as ordinary OCP user trough test service account", func() {
+			BeforeEach(func() {
+				tests.SkipIfNoCmd("oc")
```

Comment:
In this PR you replaced "kubectl" with k8sClient. Why didn't you do the same with "oc"?

Reply:
Because these tests are run specifically for OCP cluster. They do not run when there is only kubectl.